types/mem-fs/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for mem-fs 1.1
+// Project: https://github.com/sboudrias/mem-fs#readme
+// Definitions by: My Food Bag <https://github.com/MyFoodBag>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { EventEmitter } from 'events';
+import { Transform } from 'stream';
+import * as File from 'vinyl';
+
+export function create(...args: any[]): memFs.Store;
+
+export namespace memFs {
+    interface Store extends EventEmitter {
+        add: (file: File, content: string) => void;
+        each: (callback: (file: File, index: number) => void) => void;
+        get: (filepath: string) => File;
+        stream: () => Transform;
+    }
+
+    const prototype: {
+    };
```

Comment:
It seems the `prototype` can be removed

Reply:
@wrumsby you can go ahead and remove this line.